fdbserver/ClusterController.actor.cpp

```diff
@@ -619,7 +619,7 @@ void checkBetterSingletons(ClusterControllerData* self) {
 	}
 
 	WorkerDetails newEKPWorker;
-	if (SERVER_KNOBS->ENABLE_ENCRYPTION) {
+	if (SERVER_KNOBS->ENABLE_ENCRYPTION || g_network->isSimulated()) {
```

Comment:
note: `unitests` today aren't `simulation` compatible, hence, the code would not be active for `unitests` . May be it is OK, but wanted to highlight this fact.

Reply:
Good to know. With unit test I will be setting the ENABLE_ENCRYPTION knob explicitly, but I haven't add any unit tests yet.